pygotham/talks/models.py

```diff
@@ -90,7 +143,10 @@ class Talk(db.Model):
     )
 
     user_id = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
```

Comment:
@dirn On that note, it may be useful to have a `primary_user` property, e.g. to determine if a user is able to edit a talk.